app/controllers/autosign_controller.rb

```diff
@@ -3,13 +3,16 @@ def index
     @proxy = SmartProxy.authorized(:view_smart_proxies_autosign).find(params[:smart_proxy_id])
     setup_proxy
 
-    autosign = @api.autosign
-    @autosign = autosign.paginate :page => params[:page], :per_page => Setting::General.entries_per_page
+    @autosign = @api.autosign
```

Comment:
And is there a configuration value for no of rows? Or is this automatic detection?